src/devices/cpu/tlcs900/tmp95c061.cpp

```diff
@@ -229,6 +229,12 @@ void tmp95c061_device::device_start()
 	save_item(NAME(m_mem_start_mask));
 	save_item(NAME(m_dram_refresh));
 	save_item(NAME(m_dram_access));
+
+	m_nmi_state = CLEAR_LINE;
+	for( int i = 0; i < TLCS900_NUM_INPUTS; i++ )
+	{
+		m_level[i] = CLEAR_LINE;
+	}
```

Comment:
You really want to do this in `device_resolve_objects` to avoid annoyances when devices push output state on start.